catalog/app/components/Dialog/Prompt.tsx

```diff
@@ -82,8 +96,8 @@ export function usePrompt({ initialValue, title, onSubmit, validate }: PromptPro
   }, [])
   const close = React.useCallback(() => setOpened(false), [])
   const handleSubmit = React.useCallback(
-    (value: string) => {
-      onSubmit(value)
+    (...args: [React.FormEvent, string]) => {
```

Comment:
you can make it a little DRYer here:
```suggestion
    (...args: Parameters<typeof onSubmit>) => {
```